src/osx/cocoa/listbox.mm

```diff
@@ -648,6 +648,7 @@ long WXUNUSED(extraStyle))
 
     wxNSTableView* tableview = [[wxNSTableView alloc] init];
     [tableview setDelegate:tableview];
+    [tableview setFocusRingType:NSFocusRingTypeNone];
```

Comment:
Again, this is not really a comment on this change, but just to understand myself: how do you know which controls should use focus ring and which ones shouldn't?

Reply:
Observing them in native apps, or in what Xcode creates by default. It's generally "content" controls (aka big white areas) that indicate focus differently.

Reply:
This used to be different, but I guess it's like this on all of our supported OS versions

Reply:
this used to be different, but I guess in all our supported macOS versions it's like that now